Frameworks/Foundation/NSUndoManager.mm

```diff
@@ -143,4 +223,18 @@ - (void)setActionIsDiscardable:(BOOL)discardable {
     UNIMPLEMENTED();
 }
 
+/**
+ @Status Interoperable
+*/
+- (BOOL)canUndo {
+    return [((_NSUndoGroup*)[_undoStack lastObject])canUndo];
```

Comment:
Casts should not be necesasry.

---
In reply to: [75709894](https://github.com/Microsoft/WinObjC/pull/900#discussion_r75709894) [](ancestors = 75709894)